web/plates/js/status.js

```diff
@@ -68,7 +68,8 @@ function StatusCtrl($rootScope, $scope, $state, $http, $interval) {
 				var up_s = parseInt((uptime / 1000) % 60),
 					up_m = parseInt((uptime / (1000 * 60)) % 60),
 					up_h = parseInt((uptime / (1000 * 60 * 60)) % 24);
-				$scope.Uptime = String(((up_h < 10) ? "0" + up_h : up_h) + "h" + ((up_m < 10) ? "0" + up_m : up_m) + "m" + ((up_s < 10) ? "0" + up_s : up_s) + "s");
+					up_d = parseInt(uptime / (1000 * 60 * 60 * 24));
+				$scope.Uptime = String(up_d + "d" + ((up_h < 10) ? "0" + up_h : up_h) + "h" + ((up_m < 10) ? "0" + up_m : up_m) + "m" + ((up_s < 10) ? "0" + up_s : up_s) + "s");
```

Comment:
Ayup. @bkwny and I apparently implemented different solutions to his request to show the day counter. I kept the his formatting from d925990, and kept the modulo math from my branch to tighten up the code.